CMakeLists.txt

```diff
@@ -35,13 +35,23 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 add_subdirectory(${SDK_DIR})
 add_subdirectory(${MP_DIR})
 
+
+# Fetch google test dependencies
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/35d0c365609296fa4730d62057c487e3cfa030ff.zip
```

Comment:
Can you add a comment here why this exact URL is used here? And why we can't install the dependency in the installation phase, not the build phase?

Reply:
Change has been proposed by CI/CD team.